packages/api/package.json

```diff
@@ -20,10 +20,25 @@
   },
   "dependencies": {
     "logform": "^2.3.0",
-    "winston": "^3.3.3"
+    "winston": "^3.3.3",
+    "connect-mongo": "^4.6.0",
+    "connect-session-sequelize": "^7.1.2",
+    "cookie-parser": "^1.4.6",
+    "cookie-signature": "^1.1.0",
+    "express-session": "^1.17.2",
+    "mongodb": "^4.2.1",
+    "openid-client": "^5.0.2",
+    "passport": "^0.5.0",
+    "pg-hstore": "^2.3.4",
+    "sequelize": "^6.12.0-beta.1",
+    "sqlite3": "npm:@vscode/sqlite3@^5.0.7"
```

Comment:
> For any of the ones that are needed for the dev version of the session store, we should put those into the dev dependencies

Moved sqlite3, sequelize, pg-hstore and connect-session-sequelize to devDependencies
> Also, the sqlite3 dep looks like it installed strangely since it has that `npm:` thing in it. Maybe a reinstall of that will fix it?

Sqlite3 we’re installing as an alias to use vscode’s distribution since outstanding secvuln in sqlite3 latest version, that’s why the format looks strange